mkdocs.yml

```diff
@@ -173,6 +173,7 @@ nav:
           - 'Histogram': visualize_histogram.md
           - 'Overlay Images': visualize_ovelay_two_imgs.md
           - 'Pseudocolor': visualize_pseudocolor.md
+          - 'Colorize Label Image': visualize_colorize_label_img.md
```

Comment:
@DannieSheng we have generally been listing the functions alphabetically by the title in the table of contents (with some exceptions). Could you move this one up above 'Colorize Masks'?

Reply:
Yes of course! Thank you!